packages/paste-website/src/pages/components/radio-button-group/index.mdx

```diff
@@ -0,0 +1,257 @@
+import {DataTableIcon} from '@twilio-paste/icons/esm/DataTableIcon';
+import {CallIcon} from '@twilio-paste/icons/esm/CallIcon';
+import {SMSCapableIcon} from '@twilio-paste/icons/esm/SMSCapableIcon';
+import {EmailIcon} from '@twilio-paste/icons/esm/EmailIcon';
+import Changelog from '@twilio-paste/radio-button-group/CHANGELOG.md';
+import {Text} from '@twilio-paste/text';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  TextExample,
+  TextAndIconExample,
+  ErrorExample,
+  DisabledExample,
+} from '../../../component-examples/RadioButtonGroupExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/radio-button-group"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/radio-button-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Radio Button Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/radio-button-group"
+  storybookUrl="/?path=/story/components-radiobuttongroup--attached"
+  data={props.data}
+  description={props.pageContext.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{RadioButtonGroup, RadioButton}} language="jsx" noInline>
+  {TextExample}
+</LivePreview>
+
+## Guidelines
+
+### About Radio Button Group
+
+A Radio Button Group is a set of 2–6 mutually exclusive radio buttons. It is often used as a way to swap between different views of the same data (e.g., swapping between different chart types for a data set).
+
+### Radio Button Groups in forms
+
+Be thoughtful when using Radio Button Groups in forms. It can be difficult for users to discern which option is “selected” with only 2 options, while having too many options can be overwhelming. We recommend no more than 6 options.
+Radio Button Groups used in forms should have a label.
+
+### Radio Button Groups vs Tabs
+
+[Tabs](/components/tabs) are used to layer related pieces of information together and display one layer at a time. Use Tabs to alternate between views within the same context. Use Radio Button Groups to toggle between different functionality in a single view. Tabs replace the entire view based on the selected tab. Radio Buttons update the data within the view.
```

Comment:
Praise: so well explained here! especially love the last two sentences

Reply:
:)